.github/workflows/vim-patches.yml

```diff
@@ -41,7 +41,7 @@ jobs:
         run: |
           git checkout -b ${VERSION_BRANCH}
           nvim -V1 -es -i NONE +'luafile scripts/vimpatch.lua' +q
-          printf '::set-output name=NEW_PATCHES::%s\n' $([ -z "$(git diff)" ]; echo $?)
+          printf 'NEW_PATCHES=%s\n' $([ -z "$(git diff)" ]; echo $?) >> $GITHUB_OUTPUT
```

Comment:
```suggestion
          printf 'NEW_PATCHES=%s' $([ -z "$(git diff)" ]; echo $?) >> $GITHUB_OUTPUT
```

Reply:
No, because then `NEW_PATCHES` can never be 0 which is needed for this to work.

I think I'm just not gonna touch this any more than I need to so I don't become responsible if it breaks lol.